json/client.go

```diff
@@ -37,10 +40,15 @@ type clientResponse struct {
 
 // EncodeClientRequest encodes parameters for a JSON-RPC client request.
 func EncodeClientRequest(method string, args interface{}) ([]byte, error) {
+	val, err := rand.Int(rand.Reader, big.NewInt(int64(math.MaxInt64)))
```

Comment:
@coreydaley this section needs special attention during review.

Reply:
I am working on a similar issue with the websockets repository, swapping math/rand for crypto/rand, haven't gotten it completely working yet though: https://github.com/gorilla/websocket/pull/840/files#diff-4f427d2b022907c552328e63f137561f6de92396d7a6e8f6c2ea1bcf0db52654R186-R194

Reply:
Ah, looks like you don't have a weird test that goes with it though :) 
I think this looks fine, you are really just changing it to an `actual` random number :)
https://onlinegdb.com/k7-YE1UhC

Reply:
So, it's good to proceed as it is, right?

Reply:
Yes, i think it should be fine.